data/application_data.go

```diff
@@ -26,13 +26,13 @@ var (
 			URL:  "https://github.com/ibraginsky/book-server",
 		},
 	}
-	tackleTestapp = api.Application{
+	TackleTestapp = api.Application{
 		Name: "tackle-testapp",
 		Repository: &api.Repository{
 			Kind: "subversion",
 			URL:  "https://github.com/konveyor/tackle-testapp",
 		},
 		Binary: "io.konveyor.demo:customers-tomcat:0.0.1-SNAPSHOT:war",
 	}
```

Comment:
Hi @aufi, @sshveta 
IIUC, there is no need to provide the repository source to execute a binary mode analysis, but only Binary details (taken from pom.xml file).
This is a general tackle-testapp details that we may use for other analysis test cases.
In case that we run source-only/source+deps mode analysis, the repository should be accessed from subversion (SVN), since this is the `api.Application.Kind`